students/heetaek/westagram/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class User(models.Model):
+    name = models.CharField(max_length=30)
+    email = models.CharField(max_length=50)
+    password = models.CharField(max_length=50)
```

Comment:
이후에 암호화를 배우고나서 암호화된 비밀번호를 저장하게 됩니다. max_length 를 넉넉하게 잡아주세요.
```suggestion
    password = models.CharField(max_length=200)
```